src/main/java/de/uhd/ifi/se/decision/management/jira/quality/MetricCalculator.java

```diff
@@ -345,7 +358,10 @@ public MetricCalculator(Long projectId, ApplicationUser user, String issueTypeId
 				boolean relevant = false;
 				for (KnowledgeElement currentElement : elements) {
 					if (comment.getBody().contains(currentElement.getDescription())
-							&& currentElement.getTypeAsString() != "OTHER") {
+							&& currentElement.getTypeAsString() != "OTHER"

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use equals() to compare strings instead of '==' or '!='](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5632096)